docker/scripts/deprovision.sh

```diff
@@ -52,6 +52,11 @@ else
 	echo "Notice: found hardware_id ${hw_id} for instance ${id} in facility ${facility}"
 fi
 
+if [[ ${#disks[@]} -eq 0 ]]; then
```

Comment:
This should be moved below where the autofail() trap function is defined and enabled. Then you can set an autofail message that will be shown as the reason for the deprov failure in the API and in our Slack channels. E.g:

set_autofail_stage "ensure at least one disk is found"
(then do your disk check and exit 1 if not found)

That way, when your exit 1 is processed, the API and staff will be shown the message: "Error during ensure at least one disk is found"

Reply:
modified to keep it consistent with other message in the script